### PR TITLE
User Registration Updates

### PR DESCRIPTION
**Pull request by gerald**
Change country and region to dropdowns.
Remove region when country is not Philippines.
Fix typo.

### CODE REVIEW

File: app/views/devise/registrations/edit.html.erb [hunk 1]
```diff
@@ -36,7 +36,7 @@
   <div class="clearfix">
     <%= f.label :country %>
     <div class="input">
-      <%= f.text_field :country, :class => "xlarge" %>
+      <%= f.select :country, Carmen.countries.collect {|c| [c[0], c[0]] }, {:include_blank => true}, :onchange => "toggler_region()", :class => "xlarge" %>
```
Comment (crigor) [inline app/views/devise/registrations/edit.html.erb:39]: Typo. should be toggle_region()

Comment (gerald) [inline app/views/devise/registrations/edit.html.erb:39]: Ooops. Inupdate mo na ba o ako na? Hehe.

On Fri, Jan 13, 2012 at 8:00 PM, Christopher Rigor <
reply@reply.github.com

Comment (crigor) [inline app/views/devise/registrations/edit.html.erb:39]: Ikaw na.

**Comment by crigor** [comment]
closing. see https://github.com/crigor/bayaninhand/pull/17

**Pull request merged**